### ISSUE TITLE
openapi to zod?

### ISSUE THREAD

**Issue by navtoj** [asteasolutions/zod-to-openapi#89]
Is there a way to convert the openapi schema back to Zod?

**Comment by wiltonjr22** [comment]
you say take a .json file and turn it into a schema?

**Comment by samchungy** [comment]
Are you asking how to transform existing schema to this plugins .openapi format? That would be tricky.

**Comment by AGalabov** [comment]
@navtoj we do not provide a way to generate a zod schema from an openapi definition. What @samchungy suggested seems interesting. I'd also throw in a library that we used prior to migrating to `zod` + `zod-to-openapi` for documentation and validation. We used [express-openapi-validator](https://www.npmjs.com/package/express-openapi-validator) - it uses existing openapi definitions and creates a middleware on your express server. However we did have issue, thus we implemented the whole thing here and stuck with `zod` for validation - mainly because of its Typescript features and great syntax.
